src/js/constants/ServiceSchema.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+let SERVICE_SCHEMA = {
```

Comment:
Is this really a schema? It seems more like a SchemaForm configuration.
My point is: are we going to be able to use this same schema as a reference to perform JSON definition validation/transformations later on?